frontend/pages/auth/redirect.tsx

```diff
@@ -0,0 +1,30 @@
+import Container from '@material-ui/core/Container';
+import { useSession } from 'next-auth/client';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React from 'react';
+import Layout from '../../components/layout';
+
+const Redirect: React.FunctionComponent = () => {
+  const router = useRouter();
+  const [_session, loading] = useSession();
```

Comment:
Why not just `session`?

Reply:
Just so the linter doesn't complain about it being unused.  : - )